### ISSUE TITLE
Assignment of @xml:id to text containers

### ISSUE THREAD

**Issue by michaelnmmeyer** [erc-dharma/project-documentation#225]
EGC §7.2.1.1 defines a policy for assigning `@xml:id` to textual elements (`<p>`, `<ab>`, etc.). It needs to be revised, because it doesn't use stable identifiers. This makes cross-referencing pretty much impossible. We'll need proper cross-references for @amandinebricout's work, among else. Together with @manufrancis, we already discussed the issue somewhat.

I propose to adopt the following rules:

- The form of an `@xml:id` doesn't matter. It is left to the discretion of the editor. With one exception, however: the character `:` must not be used within it.
- Once a text container has been assigned an `@xml:id`, the `@xml:id` must not change. Modifying the contents of the container remains possible, as long as the overall text remains conceptually the same. For instance, correcting a typo or adding a new reading is fine.
- Significant modifications (including structural modifications: a `<p>` becoming an `<ab>`, etc.) require the assignment of a new `@xml:id`. This new `@xml:id` must not have been used at any point of the modification history of the file.
- For intra-file references, the `@xml:id` can be used as is. But when referring to an element in another file, it must be prefixed with the name of the other file, minus the extension, as in `DHARMA_CritEdDharmaPatanjala:dharmaPatanjala_01.55`.
- A program can be used to verify the well-formedness of `@xml:id`, identify broken links, etc., at the scale of the whole project.

It is important that we systematically use file prefixes or abbreviations when referring to elements in external files, otherwise it is impossible for me to tell which file I should look it up into.

**Comment by arlogriffiths** [comment]
@michaelnmmeyer : you opened this issue in 2023 and closed it a year later. There was never any discussion in this platform

Now @ajaniak needs to know what policy is to be implemented. 

My impression is that your point of view expressed in June 2023 started from the assumption that we have finished editions. But most of our editions made as per EGC are work in progress, so we cannot work with inflexible rules and the issue of link being broken due to modified identifiers is not very important in my opinion, because any editor of a critical edition will simply need to check the validiy of all cross references at the final stage of the editing process.

In short, does EGC 7.2.1.1 actually need to be revised or not?

**Comment by michaelnmmeyer** [comment]
The goal was to accommodate Amandine's need to refer to parts of inscriptions in such a way that references do not end up broken later on. But maintaining a working interlinking system is too much trouble, so this discussion can be ignored.

**Comment by arlogriffiths** [comment]
Thanks!
